src/Ginger/Translation.elm

```diff
@@ -268,10 +278,20 @@ htmlEN =
 {-| -}
 fromJson : Decode.Decoder Translation
 fromJson =
+    let
+        filter k v =
+            String.length k == 2 && not (String.isEmpty v)
```

Comment:
String.length is a bit of lame validation but do we really want to check if it's a valid iso639? The use of this function is pretty much limited to Ginger REST api responses.